spec/cb/models/implementations/resume/resume_spec.rb

```diff
@@ -55,7 +55,7 @@ module Models
 
         context 'and it is specifically the PerHourOrPerYear missing' do
           let(:resume_hash) do
-            { 'salaryInformation' => { 'jobTitle' => 'jerbs',
+            { 'salaryInformation' => { 'workExperienceId' => 'test1',
               'mostRecentPayAmount' => 100.00 },'userIdentifier' => 'user' }
           end
```

Comment:
This is more a personal nitpick than anything else, so feel free to ignore. When I have multi-line hashes, I prefer the format of

```ruby
{
  'somekey' => { 'a' => 'b' },
  'otherkey' => { 'x' => 'y', 'z' => 'z' }
}
```

to what you have. Again, this is a nitpick that you can safely ignore if desired.